apps/src/templates/InlineDropdownMenu.jsx

```diff
@@ -0,0 +1,143 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+import color from '@cdo/apps/util/color';
+import onClickOutside from 'react-onclickoutside';
+
+// HTML Adapted from:
+// https://www.w3.org/TR/wai-aria-practices-1.1/examples/menu-button/menu-button-actions.html
+
+// Displays an accessible dropdown menu with actions or links
+// TODO: Convert to a functional component once Radium is upgraded past 0.26.0
+// Our version of Radium doesn't support useState, but it has been patched in
+// later versions
+export const InlineDropdownMenu = class InlineDropdownMenuComponent extends Component {
+  static propTypes = {
+    icon: PropTypes.string.isRequired,
+    children: props => {
+      React.Children.map(props.children, child => {
+        if (child.type !== 'a') {
+          throw new Error('only accepts children of type <a/>');
+        }
+        if (!child.props.href && !child.props.onClick) {
+          throw new Error('each child must have an href or onclick');
+        }
+      });
+    }
+  };
+
+  state = {
+    isOpen: false
+  };
+
+  handleClickOutside = () => {
+    if (this.state.isOpen) {
+      this.setState({isOpen: false});
+    }
+  };
+
+  selectOptionWrapper = selectAction => {
+    this.setState({isOpen: false});
+    selectAction();
+  };
+
+  handleKeyDown = (event, selectAction) => {
+    switch (event.which) {
+      // 13 is "enter" and 32 is "space"
+      case 13:
+      case 32: {
+        this.selectOptionWrapper(selectAction);
```

Comment:
there aren't any other cases right now, but should we still add a `break` statement after this line for future usage?

Reply:
Will do. Thanks!